File: cf_sdk/src/main/java/com/example/cf_sdk/logic/changebanksdk/model/account/AchTransferStatus.java
package com.example.cf_sdk.logic.changebanksdk.model.account;

import com.google.gson.annotations.SerializedName;


public enum AchTransferStatus {
    @SerializedName("CREATED")
    CREATED("Created"),

    @SerializedName("PENDING")
    PENDING("Pending"),

    @SerializedName("SETTLED")
    SETTLED("Settled"),

    @SerializedName("FAILED")
    FAILED("Failed"),

    @SerializedName("CANCELLED")
    CANCELLED("Cancelled"),

    @SerializedName("Approved")
    Approved("Approved"),

    @SerializedName("Reversed")
    Reversed("Reversed");

    String mStatusName;

    AchTransferStatus(String statusName) {
        mStatusName = statusName;
    }

    public String getStatusName() {
        return mStatusName;
    }
}

File: cf_sdk/src/main/java/com/example/cf_sdk/changebankapi/model/account/AchTransferStatus.kt
package com.example.sdk_no_dagger.changebankapi.model.account

import com.google.gson.annotations.SerializedName

enum class AchTransferStatus(statusName: String) {
    @SerializedName("CREATED")
    CREATED("Created"), @SerializedName("PENDING")
    PENDING("Pending"), @SerializedName("SETTLED")
    SETTLED("Settled"), @SerializedName("FAILED")
    FAILED("Failed"), @SerializedName("CANCELLED")
    CANCELLED("Cancelled"), @SerializedName("Approved")
    Approved("Approved"), @SerializedName("Reversed")
    Reversed("Reversed");

    var statusName = ""

    init {
        this.statusName = statusName
    }
}